scripts/utils/update_package.py

```diff
@@ -205,6 +241,49 @@ def update_version_url(package):
     return latest_version
 
 
+def update_msixbundle_url(package):
+    install_script_path, content = get_install_script(package)
+
+    if not content:
+        return None
+
+    # Match urls like:
+    # - https://windbg.download.prss.microsoft.com/dbazure/prod/1-2502-25002-0/windbg.msixbundle
+    rex = re.search(
+        r"[\"'](https{0,1}:\/\/.+?[A-Za-z\-_]+\/(\d+\-\d+\-\d+\-\d+)\/[A-Za-z\-_]+\.msixbundle)[\"']", content
```

Comment:
[nitpick] I think using a named group increases readability:
```suggestion
        r"[\"'](?P<url>https{0,1}:\/\/.+?[A-Za-z\-_]+\/(?P<version>\d+\-\d+\-\d+\-\d+)\/[A-Za-z\-_]+\.msixbundle)[\"']", content
```